### ISSUE TITLE
Prerequsites

### ISSUE THREAD

**Issue by dnkennedy** [edickie/compare_surf_tools#3]
Hi, I was trying to run the double_check.Rmd file, and it turns out that there are some dependencies that perhaps we should educate our users about.  Specifically, I had to do the following to make everything work:
> install.packages("dplyr")
> install.packages("tidyr") (where I chose the precompiled 'older version')
> install.packages("knitr")
> install.packages("rmarkdown")

Also, when trying to knitr to a .pdf, I was required to install LaTex.

**Comment by edickie** [comment]
Yeah, maybe we should update the README with R packages used.. will do that now..

I don't think we should have problems with package versions here.. The newest dplyr, tidyr and ggplot2 now come packaged together inside the "tidyverse" can be installed with: 
```
install.packages(tidyverse)
```

rmardown and kntir come packaged within Rstudio, so anybody using it should have them already.. (and can knit using the button on the IDE. Also knitting to .html is works out of the box. It just takes up more diskspace in the repo.. 

I'll add these notes to the README.

**Comment by dnkennedy** [comment]
Note that perhaps I'm not looking at the official version of this project, which is perhaps: https://github.com/companat/compare-surf-tools

Tho I think the same issue applies there.  

So, procedurally (once I confirm that the other repo is the official) I should close this issue, and re-post it over in the other?

Or, we'll just use this issue to support the update on the real repo, and then just close it out here.

**Comment by edickie** [comment]
Yeah I was about to say that..should we try pull my version into the
central version now??

**Comment by dnkennedy** [comment]
So, it seem that you and steve have updates; I'll let you githubers figure out how to merge all this in the main official repo.

**Comment by edickie** [comment]
figuring it out now..
